src/crl.c

```diff
@@ -757,11 +757,7 @@ int BufferLoadCRL(WOLFSSL_CRL* crl, const byte* buff, long sz, int type,
     int          ret = WOLFSSL_SUCCESS;
     const byte*  myBuffer = buff;    /* if DER ok, otherwise switch */
     DerBuffer*   der = NULL;
-#ifdef WOLFSSL_SMALL_STACK
-    DecodedCRL*  dcrl;
-#else
-    DecodedCRL   dcrl[1];
-#endif
+    WC_DECLARE_VAR(dcrl, DecodedCRL, 1, 0);
```

Comment:
I realize you are using an existing macro with `WC_DECLARE_VAR`, but perhaps we should remove the heap arg since its not used in either case and is always 0 (NULL)?

Reply:
i tried that, but it messes with the FIPS code. see https://github.com/wolfSSL/wolfssl/pull/9153#issuecomment-3253574414